docs/content/DimensionSpecs.md

```diff
@@ -154,3 +154,27 @@ Example for the `__time` dimension:
   "function" : "function(t) { return 'Second ' + Math.floor((t % 60000) / 1000); }"
 }
 ```
+
+### Explicit lookup extraction function
+Explicit lookups allow you to specify a set of keys and values to use when performing the extraction
+```json
+{
+  "type":"explicitLookup",
+  "lookup":{"foo":"bar", "baz":"bat"},
+  "retainMissingValue":true,
+  "uniqueLookups":true
+}
+```
+
+```json
+{
+  "type":"explicitLookup",
+  "lookup":{"foo":"bar", "baz":"bar"},
+  "retainMissingValue":false,
+  "uniqueLookups":false,
+  "replaceMissingValueWith":"MISSING"
+}
+```
+A property of `retainMissingValue` and `replaceMissingValueWith` can be specified at query time to hint how to handle missing values. Setting `replaceMissingValueWith` to `""` has the same effect of setting it to `null` or omitting the property. Setting `retainMissingValue` to true will use the dimension's original value if it is not found in the namespace. The default values are `replaceMissingValueWith = null` and `retainMissingValue = false` which causes missing values to be treated as missing.
```

Comment:
If `retainMissingValue=true` and `replaceMissingValueWith=billy`, which will take precedence?

Also, you say "if it is not found in the namespace", I think that should be "if it is not found in the lookup" given that we are calling this a lookup now.